DESCRIPTION

```diff
@@ -23,13 +23,14 @@ Imports:
     cli,
     desc (>= 1.2.0),
     enc,
-    gert (>= 1.0.0),
+    gert (>= 1.3.9000),
```

Comment:
Needed for now because `git_branch_move()` is brand-new.

Reply:
Let's hope it makes it to CRAN before we release.

Reply:
It made it to CRAN yesterday :tada: